src/services/codefixes/convertToEs6Module.ts

```diff
@@ -115,7 +129,7 @@ namespace ts.codefix {
                     }
                     case SyntaxKind.BinaryExpression: {
                         const { operatorToken } = expression as BinaryExpression;
-                        return operatorToken.kind === SyntaxKind.EqualsToken && convertAssignment(sourceFile, checker, expression as BinaryExpression, changes, exports);
+                        return operatorToken.kind === SyntaxKind.EqualsToken && convertAssignment(sourceFile, checker, expression as BinaryExpression, changes, exports, useSitesToUnqualify);

```

Comment:
is a change to convertAssignment already in this PR? Or did it already take a map of use sites to unqualify?

Reply:
It’s here, GitHub folded over it: https://github.com/microsoft/TypeScript/pull/40987/files#diff-d91b63cfe65f0adf550d61fede06a78dR213